docs/filters.rst

```diff
@@ -0,0 +1,87 @@
+.. _filters:
+
+Filtering
+=========
+
+REST resource filtering is provided via method ``_filter_queryset``. If you want implement your own filter technique
+you can add here your own code. Input params of the method are only ``qs`` which represents data that you want to filter
+(for model resources it is django QuerySet object).
+
+Better way is used pre-implemented managers defined in model ``BaseModelResource``::
+
+    class BaseModelResource(DefaultRESTModelResource, BaseObjectResource):
+        filter_manager = MultipleFilterManager()
+
+        def _filter_queryset(self, qs):
+            if self.filter_manager:
+                return self.filter_manager.filter(self, qs, self.request)
+            else:
+                return qs
+
+There method ``_filter_queryset`` uses manager to filter output data. You can change manager with filter_manager
+attribute. If you set filter_manager to ``None``. Filtering will be disabled. Default filter manager is
+``MultipleFilterManager``.
+
+Filter manager
+--------------
+
+The main purpose of filter manager is parse input data, construct django models Q objects and use it to filter data
+from model QuerySet. Manager has only one public method ``filter(self, resource, qs, request)`` that accepts
+resource object, QuerySet and Django request object.
```

Comment:
The main purpose of a filter manager is to parse the filter specification, construct Django Q objects and use it to filter the data in the QuerySet.
Manager has only one public method ``filter(self, resource, qs, request)``. 
It accepts a resource object, a QuerySet to filter and a Django request object containing the filter specification in GET arguments.